BookStoreProd/src/main/java/mate/academy/bookstoreprod/controller/BookController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.bookstoreprod.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstoreprod.dto.BookDto;
+import mate.academy.bookstoreprod.dto.CreateBookRequestDto;
+import mate.academy.bookstoreprod.service.BookService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/books")
```